docs/source/recipes/torch-dataset-examples/utils.py

```
@@ -8,6 +8,31 @@
 
+
+# Helper class to wrap a function with field names for vectorized caching
+class SimpleGetItem(GetItem):
+    """A simple wrapper that allows using a function with specific field names for caching.
+
+    Args:
+        func: A callable that takes a dict of field_name -> value and returns the processed data
+        field_names: A list of field names to cache from the dataset
+    """
+
+    def __init__(self, func, field_names):
```

Ruff (0.14.7)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)